agent/spec/lib/kontena/launchers/etcd_launcher_spec.rb

```diff
@@ -94,14 +98,14 @@
       allow(container).to receive(:running?).and_return(true)
       allow(container).to receive(:info).and_return({'Config' => {'Image' => 'etcd'}})
       expect(subject.wrapped_object).to receive(:add_dns)
-      node_info = {
+      node = Node.new(
```

Comment:
I sense a great conflict with #1323 :(

It refactors this spec: https://github.com/kontena/kontena/pull/1323/files#diff-88ba28942ba5ccb7c85e35a737d4f14f